### PR TITLE
Kmr web-manager

### PR DESCRIPTION
**Pull request by comzyh**
image

### CODE REVIEW

File: manager/manager.go [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+	log.Infof("Listening at %s", *port)
+	server.server.ListenAndServe()
+	select {}
+}
+
+func (server *KmrManagerWeb) IndexHandler(w http.ResponseWriter, r *http.Request) {
+	http.ServeFile(w, r, path.Join(server.uiDir, "index.html"))
+}
+
+func (server *KmrManagerWeb) HelloHandler(w http.ResponseWriter, r *http.Request) {
```
Comment (Lcch) [inline manager/manager.go]: what is this API for?

File: manager/manager.go [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+	nameRegex, _ := regexp.Compile("^[a-z]([-a-z0-9]*[a-z0-9])?\\.json$")
+	result := make(map[string]string)
+
+	files, _ := ioutil.ReadDir(directory)
+	for _, file := range files {
+		if !nameRegex.Match([]byte(file.Name())) {
+			continue
+		}
+		filename := file.Name()
+		name := filename[:len(filename)-len(".json")]
```
Comment (Lcch) [inline manager/manager.go]: please make sure len(filename) > len(".json").

Comment (comzyh) [inline manager/manager.go]: the regexp 

> ^[a-z]([-a-z0-9]*[a-z0-9])?\\.json$

have already deal with it.

File: manager/manager.go (hunk 1 above)
Comment (comzyh) [inline manager/manager.go]: I will delete it

**Pull request merged**